lib/src/ast/sass/statement/use_rule.dart

```diff
@@ -31,6 +33,14 @@ class UseRule implements Statement {
       {Map<String, Tuple2<Expression, FileSpan>> configuration})
       : configuration = Map.unmodifiable(configuration ?? const {});
 
+  /// Parses a variable declaration from [contents].
```

Comment:
This should say "use rule" instead of "variable declaration"

Reply:
Done.